### PR TITLE
fix(web-client): diasble autologout in send & pull pay

### PR DESCRIPTION
**Pull request by IscoRuta98**
Aim of this PR is to disable Auto Logout when user is confirming transactions (for pull and pay payments).

### CODE REVIEW

**Comment by billguo99** [comment]
Have you tested this locally?

File: web-client/src/app/views/pay/pay.page.ts [hunk 1]
```diff
@@ -145,19 +148,22 @@ export class PayPage implements OnInit {
         }
```
Comment (billguo99) [inline web-client/src/app/views/pay/pay.page.ts:148]: Do we need to add `this.autoLogoutService.enableAutoLogout()` in the catch (error) block above on line 144?

Comment (IscoRuta98) [reply web-client/src/app/views/pay/pay.page.ts:148]: No, it is not needed, will delete it in the latest commit.

Comment (billguo99) [reply web-client/src/app/views/pay/pay.page.ts:148]: Wait, I think it's needed? I meant another catch block.
I saw you added the enableAutoLogout in the catch block and then I saw another one where you didn't add it.

Comment (IscoRuta98) [reply web-client/src/app/views/pay/pay.page.ts:148]: I don't think so as the one I deleted should not have been added in the catch block as it (enableAutologout() method) was already added on the try block.

**Review by billguo99 — APPROVED** [review]
Cool!

**Pull request merged**